src/Samples/NetCoreServer/Program.cs

```diff
@@ -7,6 +8,29 @@ namespace NetCoreServer
 {
     class Program
     {
+        /*
+        static void Main(string[] args)
```

Comment:
A lot of cleanup needed in the sample code

Reply:
I will not check in... I posted part of debugging/testing to understand framework/core behaviors in scenarios..